src/main/resources/bidder-config/generic.yaml

```diff
@@ -4,6 +4,17 @@ adapters:
     aliases:
       genericAlias:
         enabled: false
+      blue:
+        enabled: false
+        endpoint: https://prebid-us-east-1.getblue.io/?src=prebid
```

Comment:
This is a regional endpoint. Is this correct?

Reply:
Seems like yes

https://github.com/prebid/prebid-server/blob/64c5e35af385b0999e3c534dcb8ebaf7af15d846/static/bidder-info/blue.yaml#L3